kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/api/common/LocalRef.java

```diff
@@ -46,6 +44,10 @@ public final boolean equals(Object obj) {
                 && Objects.equals(getName(), other.getName());
     }
 
+    public NamespacedRef<T> inNamespaceOf(HasMetadata resource) {
```

Comment:
Javadoc

Reply:
done (on the new static method in NamespacedRef since I've moved it out of the common package)